Description: fdo-desktop-keywords
 Add freedesktop.org keywords to desktop file.

--- partitionmanager-3.1.0.orig/src/org.kde.partitionmanager.desktop
+++ partitionmanager-3.1.0/src/org.kde.partitionmanager.desktop
@@ -160,4 +160,4 @@ Icon=partitionmanager
 Categories=System;Filesystem;KDE;Qt;
 
 X-KDE-StartupNotify=true
-
+Keywords=partition;disk;
